libse/StringExtensions.cs

```diff
@@ -185,5 +186,17 @@ public static string RemoveControlCharactersButWhiteSpace(this string s)
             return sb.ToString();
         }
 
+        public static string CapitalizeFirstLetter(this string s, CultureInfo ci = null)
+        {
+            var si = new StringInfo(s);
+            if (ci == null)
+                ci = CultureInfo.CurrentCulture;
+            if (si.LengthInTextElements > 0)
+                s = si.SubstringByTextElements(0, 1).ToUpper(ci);
+            if (si.LengthInTextElements > 1)
+                s += si.SubstringByTextElements(1);
```

Comment:
Hmm... what if s is already uppercase?
`s += si.SubstringByTextElements(1).ToLower(ci);`?